libafl/src/fuzzer/mod.rs

```diff
@@ -159,6 +159,21 @@ where
         send_events: bool,
     ) -> Result<(ExecuteInputResult, Option<CorpusId>), Error>;
 
+    /// Evaluates an executed input exit kind and triggers observers and feedback.
+    /// This can be used e.g. when the input was executed but the result still needs to be evaluated and
+    /// possibly added to the corpus.
+    /// Adds an input, to the corpus even if it's not considered `interesting` by the `feedback`.
+    /// Returns the `index` of the new testcase in the corpus.
+    /// Usually, you want to use [`Evaluator::evaluate_input`], unless you know what you are doing.
+    fn evaluate_input_exit_kind(
```

Comment:
I don't like the name too much, can we find a more descriptive one?

Reply:
I am also not entirely sold on this one, but could not come up with something more fitting. Because it evaluates the input based on the exit code. 
If someone has a better idea, i can change that one